napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -841,11 +915,14 @@ def _install_packages(self, packages: Sequence[str] = ()):
         if packages:
             self.installer.install(packages)
 
-    def _handle_yield(self, project_info):
+    def _handle_yield(self, data):
```

Comment:
could you add a type hint to data?  it's what got me noticing that the type_hints from the iter_x functions were wrong

Reply:
Will do!